.gitsubprojects

```diff
@@ -6,4 +6,4 @@ git_subproject(Pression https://github.com/Eyescale/Pression.git 08cd4bf)
 git_subproject(hwsd https://github.com/Eyescale/hwsd.git 03b6581)
 git_subproject(Collage https://github.com/Eyescale/Collage.git e2bc13d)
 git_subproject(GLStats https://github.com/Eyescale/GLStats.git f9bf19e)
-git_subproject(Deflect https://github.com/BlueBrain/Deflect.git 9373b53)
+git_subproject(Deflect https://github.com/BlueBrain/Deflect.git cc4c732)
```

Comment:
Why not 4bc4119?

Reply:
I tired to explain it in the commit message. Between 0.14.0 and 0.14.1 we started using C++14 in Deflect. But Equalizer and its dependencies is still using C++11. I thought it best to keep the change minimal rather than require a switch to C++14 in Equalizer.

Reply:
My fault, I didn't read it...